src/features/expense/ui/DayExpensesItem/DayExpensesItem.tsx

```
@@ -0,0 +1,23 @@
+  return (
+    <div id={id}>
+      <p>{day.toISOString()}일</p>
+      <ul>
+        {expenses.map(({ id, category, memo, amount }: Expense) => (
+          <ExpenseItem
+            id={id}
+            category={category}
+            amount={amount}
+            memo={memo}
+          />
+        ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)